css/index.css

```diff
@@ -1,3 +1,255 @@
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, info, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, info, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* / Set every element's box-sizing to border-box / */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* / Main / */
+
+.container {
+	width: 876px;
+	margin: 0 auto;
+}
+
+/* / Header / */
+
+header {
+	margin: 30px 0;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.main-nav {
+	margin-top: 10px;
+}
+
+.main-nav a {
+	text-decoration: none;
+	color: #adadad;
+	padding-right: 50px;
+	margin-top: 15px;
+}
+
+
+/* / Call To Action / */
+
+.main-cta {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-around;
+	margin: 40px 0px; 
+}
+
+.cta {
+	align-self: center;
+}
+
+.cta-heading {
+	font-size: 70px;
+	text-align: center;
+} 
+
+.button {
+	margin-left: 90px;
+	text-decoration: none;
+	color: #000;
+	border: 1px solid #000;
+	padding: 0px 20px;
+}
+
+/* / Features / */
+
+.features-about {
+	display: flex;
+	flex-direction: row; 
+	justify-content: flex-start;
+	margin: 30px 0px;
+}
+
+.features {
+	margin-right: 20px;
+}
+
+/* / Services / */
+
+.services-info {
+	display: flex;
+	flex-direction: row; 
+	justify-content: flex-start;
+	margin: 30px 0px;
+}
+
+.services, .product {
+	padding-right: 20px;
+}
+
+
+/* / Contact / */
+
+.contact {
+	margin: 40px 0px;
+}
+
+/* / Footer / */
+
+footer {
+	display: flex;
+	justify-content: center;
+	margin-bottom: 100px;
+}
+
+/* 
+	Begin Media Queries
+	-----------------------
+	Tablet
+*/
+
+@media (max-width:800px) {
+	.container {
+		width: 750px;
+	}
+
+	/* / Header Styles / */
+	header {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		align-content: space-evenly;
+	}
+
+	.logo {
+		order: -1;
+		width: 20%;
+		margin-bottom: 20px;
+	}
+
+	.main-nav a {
+		margin-bottom: 20px;
+		margin-left: 20px;
+	}
+
+	/* / Call to Action Styles / */
+	.cta {
+		margin: 30px 0px;
+		margin-left: 150px;
+	}
+
+	.CTA-image {
+		display: none;
+	}
+
+	/* / Mid Section / */
+	.middle-img {
+		width: 100%;
+	}
+}
+
+/* / Mobile / */
+
+@media (max-width:500px) {
+	.container {
```

Comment:
good job using the containers to do a lot of the work for you, makes you have to work a lot less. I like it.